addons/mass_mailing/models/mail_mail.py

```diff
@@ -68,7 +68,7 @@ def _get_unsubscribe_url(self, cr, uid, mail, email_to, msg=None, context=None):
                 'params': urllib.urlencode({'db': cr.dbname, 'res_id': mail.res_id, 'email': email_to})
             }
         )
-        return '<small><a href="%s">%s</a></small>' % (url, msg or 'Click to unsubscribe')
+        return url, msg
```

Comment:
Why changing the return style of this one, especially that there are other methods always returning html content in the mail post processing ?